site/src/_layouts/default.hbs

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <title>d3fc {{#if title}}:{{/if}} {{title}}</title>
+    <meta name="description" content="A collection of components that make it easy to build interactive charts with D3.">
+
+    <script src="{{assets}}/scripts.js"></script>
+
+    <link rel="stylesheet" href="{{assets}}/styles.css" />
+  </head>
```

Comment:
I didn't have any specific features in mind; I had a couple of thoughts though:
- I think some of Bootstrap's features might require HTML5 polyfills to work properly in older browsers - I'm not certain though
- SVG support detection - if a browser doesn't support SVG, it would be nice to show a message to the user and load a `png` (or similar) logo instead